shelley/chain-and-ledger/executable-spec/src/STS/Ocert.hs

```diff
@@ -66,10 +66,17 @@ ocertTransition = do
       s = bheaderSlotNo bhb
   kp@(KESPeriod kp_) <- liftSTS $ kesPeriod s
 
-  let t = kp_ - c0_
+  maxKESiterations <- liftSTS $ asks maxKESEvo
 
   c0 <= kp ?! KESBeforeStartOCERT
-  kp_ < c0_ + 90 ?! KESAfterEndOCERT
+  kp_ < c0_ + (fromIntegral maxKESiterations) ?! KESAfterEndOCERT
```

Comment:
at some point we should rename `KESPeriod` to something like `Day`. doesn't have to be now, though.